docs/pipelines/shiva.md

```diff
@@ -2,92 +2,87 @@
+ -cv,--config_value <config_value>     Config values, value should be formatted like 'key=value' or 
+                                       'namespace:namespace:key=value'
+ -DSC,--disablescatter                 Disable all scatters
 
 ~~~
 
 To run the pipeline:
 ~~~
-biopet pipeline shiva -config MySamples.json -config MySettings.json -run
+biopet pipeline shiva -config MySamples.yml -config MySettings.yml -run
 ~~~
 
 A dry run can be performed by simply removing the `-run` flag from the command line call.
 
-[Gears](gears) is run automatically for the data analysed with `Shiva`. There are two levels on which this can be done and this should be specified in the [config](../general/config) file:
+[Gears](gears.md) is run automatically for the data analysed with `Shiva`. There are two levels on which this can be done and this should be specified in the [config](../general/config) file:
 
 *`mapping_to_gears: unmapped` : Unmapped reads after alignment. (default)
 *`mapping_to_gears: all` : Trimmed and clipped reads from [Flexiprep](flexiprep).
 *`mapping_to_gears: none` : Disable this functionality.
 
-### Only variant calling
-
-It is possible to run Shiva while only performing its variant calling steps.
-This has been separated in its own pipeline named `shivavariantcalling`.
-As this calling pipeline starts from BAM files, it will naturally not perform any pre-processing steps.
-
-To view the help menu, execute:
-~~~
-java -jar </path/to/biopet.jar> pipeline shivavariantcalling -h
-
-Arguments for ShivaVariantcalling:
- -BAM,--inputbams <inputbams>          Bam files (should be deduped bams)
- -sample,--sampleid <sampleid>         Sample ID (only effects summary and not required)
- -library,--libid <libid>              Library ID (only effects summary and not required)
- -config,--config_file <config_file>   JSON config file(s)
- -DSC,--disablescatter                 Disable all scatters
-
-~~~
-
-To run the pipeline:
-~~~
-biopet pipeline shivavariantcalling -config MySettings.json -run
-~~~
-
-A dry run can be performed by simply removing the `-run` flag from the command line call.
-
-
+An example of MySettings.yml file is provided here and more detailed config options are explained in [config options](#config-options).
+``` yaml
+samples:
+    SampleID:
+        libraries:
+            lib_id_1:
+                bam: YourBam.bam
+            lib_id_2:
+                R1: file_R1.fq.gz
+                R2: file_R2.fq.gz
+species: H.sapiens
+reference_name: GRCh38_no_alt_analysis_set
+dbsnp_vcf: <dbsnp.vcf.gz>
+vcffilter:
+    min_alternate_depth: 1
+output_dir: <output directory>
+variantcallers:
+    - haplotypecaller
+    - unifiedgenotyper
+    - haplotypecaller_gvcf
+unifiedgenotyper:
+    merge_vcf_results: false # This will do the variantcalling but will not merged into the final vcf file
+```
 ----
 
-## Variant caller
+## Supported variant callers
 At this moment the following variant callers can be used
```

Comment:
This should be a table with the following items as config value:
- haplotypecaller_gvcf
- haplotypecaller_allele
- unifiedgenotyper_allele
- unifiedgenotyper
- haplotypecaller
- freebayes
- raw
- bcftools
- bcftools_singlesample
- varscan_cns_singlesample

Currently it's unclear what each config value is for each caller